content/documents/configuring-user-application-data-access.mdx

```diff
@@ -0,0 +1,502 @@
+  console.log("Assignment successful!");
+})
+.catch((ex: OrderCloudError) => console.log(ex));`,
+csharp: `using OrderCloud.SDK; \n
+var data = new SecurityProfileAssignment
+{
+  "SecurityProfileID": "defaultbuyer",
+  "BuyerID": "buyeruser"
+};
+try {
+  await SecurityProfiles.SaveAssignment(data);
+  Console.WriteLine("Assignment successful!")
+} catch(OrderCloudException ex) {
+  Console.WriteLine(ex.Message);
+}
+`
+}}
+/>
+
+Next, let's make a request to assign our `defaultseller` security profile to our sellers.  To assign a security profile to all seller users, the only thing we specify in the request body is the ID of the security profile.  If successful, you should receive a `204 Created` response.
+<CodeExample
+  content={{
+    http: `POST https://sandboxapi.ordercloud.io/v1/securityprofiles/assignments HTTP/1.1
+Authorization: Bearer INSERT_ACCESS_TOKEN_HERE
+Content-Type: application/json; charset=UTF-8\n
+{
+  "SecurityProfileID": "defaultseller"
+}`,
+javascript: `import { SecurityProfiles } from "ordercloud-javascript-sdk";\n
+SecurityProfiles.SaveAssignment({
+  "SecurityProfileID": "defaultseller"
+})
+.then(() => {
+    console.log("Assignment successful!");
+})
+.catch((ex) => console.log(ex));
+`,
+typescript: `import { SecurityProfiles, SecurityProfile, OrderCloudError } from "ordercloud-javascript-sdk";\n
+SecurityProfiles.SaveAssignment({
+  "SecurityProfileID": "defaultseller"
+})
+.then(() => {
+  console.log("Assignment successful!");
+})
+.catch((ex: OrderCloudError) => console.log(ex));`,
+csharp: `using OrderCloud.SDK; \n
+var data = new SecurityProfileAssignment
+{
+  "SecurityProfileID": "defaultseller"
+};
+try {
+  await SecurityProfiles.SaveAssignment(data);
+  Console.WriteLine("Assignment successful!")
+} catch(OrderCloudException ex) {
+  Console.WriteLine(ex.Message);
+}
+`
+}}
+/>
+
+### Make authentication requests
+Now that we have configured application data access with API Clients, user access with Security Profiles and made all necessary assignments, let's make some authentication requests to check our handy work.
+First, let's call to authenticate our buyer user against our buyer API client we created earlier.
+<CodeExample
+  content={{
+    http: `POST https://sandboxapi.ordercloud.io/v1/outh/token HTTP/1.1
+Content-Type: application/x-www-form-urlencoded;\n
+client_id=00000000-0000-0000-0000-000000000000&grant_type=password&username=buyeruser&password=Supersecurepassword123!&scope=Shopper
+`,
+javascript: `import { Auth } from "ordercloud-javascript-sdk";\n
+Auth.Login("defaultbuyer", "Supersecurepassword123!", "00000000-0000-0000-0000-000000000000", ["Shopper"])
+.then((token) => {
+    console.log(token);
+})
+.catch((ex) => console.log(ex));
+`,
+typescript: `import { Auth, AccessToken, OrderCloudError } from "ordercloud-javascript-sdk";\n
+Auth.Login("defaultbuyer", "Supersecurepassword123!", "00000000-0000-0000-0000-000000000000", ["Shopper"])
+.then((token: AccessToken) => {
```

Comment:
I believe that the response carries more than just an access token. It will return an `AccessToken` which has multiple properties - one of which is the `access_token`. https://github.com/ordercloud-api/ordercloud-javascript-sdk/blob/master/src/models/AccessToken.ts

Reply:
I realize you're using the right type here, but the response variable name is a little misleading.